src/app/components/header/header.component.scss

```diff
@@ -93,15 +95,18 @@ header {
     padding-top: 8px;
     margin-left: 10px;
     text-transform: uppercase;
-    color: white;
     text-decoration: none;
 
     &:hover,
     &:focus {
-      border-color: #ffffff61;
+      @include theme('border-color', 'accent-color-dark');
     }
   }
 
+/*****************************************************************************
+*   MOBILE
+******************************************************************************/
+
```

Comment:
Not everything down is mobile, `.action-button` is not.
Also I think `mobile-menu-toggle` is self descriptive enough and we don't need a separator.

Reply:
Can you double check this? Maybe I am misreading but it looks like everything below the separator is part of either mobile-menu-toggle and mobile-menu classes. There is also an action-button subclass under the header class above this.